specification/msi/resource-manager/Microsoft.ManagedIdentity/preview/2015-08-31-preview/ManagedIdentity.json

```diff
@@ -34,6 +26,45 @@
     }
   },
   "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{resourceProviderScope}/providers/Microsoft.ManagedIdentity/identities/default": {
```

Comment:
The scope "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{resourceProviderScope}" isn't fully correct because we support resources at other levels. It think this was correct before where you had `{scope}/providers/Microsoft.ManagedIdentity/identities/default`

Reply:
I was taking it from that list. But if we do support it at other levels, then I can rollback.
https://msazure.visualstudio.com/Unified%20Platform%20KPIs/_workitems/edit/3598616

Reply:
We do support it at other levels. 👍 for rollback